laa-portal.cert

```diff
@@ -0,0 +1 @@
+MIIDODCCAiCgAwIBAgIJAJSRXNZd/kGVMA0GCSqGSIb3DQEBBQUAMBwxGjAYBgNVBAoTEUxBQSwgQ049UG9ydGFsLVNQMB4XDTE4MDEyNTE2MjE1NloXDTI4MDEyNTE2MjE1NlowHDEaMBgGA1UEChMRTEFBLCBDTj1Qb3J0YWwtU1AwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDRa4Yfdy7Isbb58LZ4GsXQiDDJIaVXrYHsda3PcKjN8h6svGDSNjWmk4+Q4cCQHPB92EKkHsI7uqpIDooIXeS4Ko71hGsfy+/H2UC0nrqptoTZLNK0tTRBpjbmdhtz5giM919N6RfKt2rrw577XzqOKzshiomFu08mM9pP/OPXmGQm5D4H4g4Y7u8hkBqg8mv48hYJxLLRnl9Y1JugkVDA5Mm4V/GRMS6ok6kGa2uWitA8keRlfRx6xsFgNluay1uosDo2tjQBwBXyaVO9dfbJDxceKKz3wlJby6Ma5i45i41WXXFagSnEg89mCKL/AP7QyJQGgTQo0eeSycK0XNv/AgMBAAGjfTB7MB0GA1UdDgQWBBRdWLZVn0h0Lla/y3Cn1Rm2gswARTBMBgNVHSMERTBDgBRdWLZVn0h0Lla/y3Cn1Rm2gswARaEgpB4wHDEaMBgGA1UEChMRTEFBLCBDTj1Qb3J0YWwtU1CCCQCUkVzWXf5BlTAMBgNVHRMEBTADAQH/MA0GCSqGSIb3DQEBBQUAA4IBAQBA75RB+4NXKbOV5pxpz6ZGv6rxZbCVSqq5yowyQFFw5jv6W/Ga3Rfumgl4On53cKrzhRuDLbneNrjvcB0rQRSnBms0X/Otc8asXABLOf4W3ONKPn5O0jd5En8D+SauolCfpfomcRTj6XxxQe6NaIgoWf9SkKiDxWy7AYQnn6Y+c4E+r0lLcYKmC+GXKyzCjdEj418f0Zflh3+nklsK/oNtpC8+K/h568TQHEWCIbMjYQ0T5ai8M9LmqGSFFH3ScQlWx++93rWOfAT+jF4EOrkL0aFULyatalAnUuJNhA1W1Fk8qUZiOTQKlUuYu/4IyY2t6YHBKgId0r0pfXekCPBD
```

Comment:
Should this be in Git?

Reply:
Speaking with @bleach about this, the idea I got is that this is the public certificate and therefore there's no security implications. I think the fact that https://samlmock.dev.legalservices.gov.uk/metadata exposes that same information also re-enforced that idea.

Reply:
Fair enough. Thought it worth checking.